uiTestHelpers/mobileStepDefinitions/checks/checkElementInsideElementInputValue.js

```diff
@@ -1,11 +1,10 @@
-// module.exports = function checkElementInsideElementInputValue(locatorKey2, locatorKey, isNot, expectedVal) {
-//   const expectedValue = expectedVal === undefined ? '' : expectedVal;
+module.exports = async function checkElementInsideElementInputValue(locatorKey2, locatorKey, isNot, expectedVal) {
```

Comment:
For consistency, shouldn't the checkElementInsideElement function take the arguments in the same order as getElementInsideElement? Like key1, key2

Reply:
see other comment

Reply:
see other comment